README.md

```diff
@@ -29,9 +29,19 @@ In order to get started quickly with Fides, a sample project is bundled within t
 
 #### Download and install Fides
 
-You can easily download and install Fides using `pip`. Run the following command to get started:
+> [!TIP]
```

Comment:
Thanks, I like this! I edited the suggested steps slightly but this `[!TIP]` treatment is a nice way to visually mark it as optional